package grab.bit.android.pages.batchdownload

import grab.bit.shared.pages.batchdownload.BaseBatchDownloadComponent
import com.arkivanov.decompose.ComponentContext

class AndroidBatchDownloadComponent(
    ctx: ComponentContext,
    onClose: () -> Unit,
    importLinks: (List<String>) -> Unit,
) : BaseBatchDownloadComponent(
    ctx = ctx,
    onClose = onClose,
    importLinks = importLinks
) {
    sealed interface Effects : BaseBatchDownloadComponent.Effects.PlatformEffects {
        // nothing for now
    }
}
